scalafix-docs/src/main/scala/docs/website.scala

```diff
@@ -23,32 +23,30 @@ package object website {
       .filterNot(_.name.isDeprecated)
       .sortBy(_.name.value)
 
-    val rows: List[Text.TypedTag[String]] = rules.map { rule =>
-      val docPath = PathIO.workingDirectory
-        .resolve("docs")
-        .resolve("rules")
-        .resolve(rule.name.value + ".md")
-      if (!Files.isRegularFile(docPath.toNIO)) {
-        reporter.warning(s"Missing $docPath")
-      }
-      val semantic = if (rule.isInstanceOf[v1.SemanticRule]) "✅" else ""
-      tr(
-        td(semantic),
-        td(ruleLink(rule.name.value)),
-        td(rule.description)
-      )
-    }
-    val html = table(
-      thead(
+    val (semantic, syntaxtic) = rules.partition(_.isInstanceOf[v1.SemanticRule])
+
+    def buildTable(rules: List[v1.Rule]) = {
```

Comment:
This method returns Unit, to fix it
```diff
diff --git a/scalafix-docs/src/main/scala/docs/website.scala b/scalafix-docs/src/main/scala/docs/website.scala
index c00761de..aca497dc 100644
--- a/scalafix-docs/src/main/scala/docs/website.scala
+++ b/scalafix-docs/src/main/scala/docs/website.scala
@@ -23,10 +23,11 @@ package object website {
       .filterNot(_.name.isDeprecated)
       .sortBy(_.name.value)
 
+    pprint.log(rules.map(_.name.value))
     val (semantic, syntaxtic) = rules.partition(_.isInstanceOf[v1.SemanticRule])
 
-    def buildTable(rules: List[v1.Rule]) = {
-      val rows: List[Text.TypedTag[String]] = rules.map { rule =>
+    def buildTable(rules: List[v1.Rule]): List[Text.TypedTag[String]] = {
+      rules.map { rule =>
         val docPath = PathIO.workingDirectory
           .resolve("docs")
           .resolve("rules")

```

After that, it renders weird

<img width="863" alt="screen shot 2018-08-24 at 17 54 53" src="https://user-images.githubusercontent.com/1408093/44594584-e14bbb00-a7c6-11e8-8cb2-29f0935f9a96.png">